### DISCUSSION TITLE
Running recommendation models on AMD CPUs

### DISCUSSION THREAD

**Discussion by rishucoding** [Xilinx/inference-server#128] [Q&A]
Hi,

I am interested in running Recommendation models like Deep Learning Recommendation Models (DLRM) on AMD CPUs. Like, I would like to run [MLPerf DLRM ](https://github.com/harvard-acc/DeepRecSys)or [DeepRecSys](https://github.com/harvard-acc/DeepRecSys). 
I am wondering if AMD provides specific guidelines/support (CPU-tuning, libraries, etc) similar to [Intel Zoo](https://github.com/IntelAI/models/blob/master/quickstart/recommendation/pytorch/dlrm/inference/cpu/README.md) for optimizing inference performance? 

Thanks,
Rishabh Jain

**Comment by varunsh-xilinx** [comment]
I've asked the ZenDNN team to comment. In the meantime, you can check the [ZenDNN documentation](https://www.amd.com/en/developer/zendnn.html) if you haven't already.

**Comment by ratan-prasad** [comment]
Since DLRM is a shallow model, we need to put multiple instances simultaneously to get optimal performance. We are in the process of creating scripts for multi-instance runs and tuning guide for optimal performance. Hopefully, this will be released with the next ZenDNN release.

**Reply by rishucoding** [reply]
Thanks @ratan-prasad. I have a few follow-up questions: 
1. What does a shallow model mean? Does it mean small in size? 
2. For putting multiple instances on a same CPU core, are you suggesting to explore various ways to deploy multi-threading?
